Configuration/StandardSequences/python/DigiToRaw_Repack_cff.py

```diff
@@ -56,3 +56,23 @@
 DigiToHybridRawRepack = cms.Sequence( DigiToHybridRawRepackTask )
 DigiToVirginRawRepack = cms.Sequence( DigiToVirginRawRepackTask )
 DigiToSplitRawRepack = cms.Sequence( DigiToRawRepackTask, DigiToVirginRawRepackTask )
+
+from EventFilter.SiStripRawToDigi.SiStripDigis_cfi import siStripDigis
+siStripDigisHLT = siStripDigis.clone(ProductLabel = "rawDataRepacker")
+
+from RecoLocalTracker.Configuration.RecoLocalTracker_cff import siStripZeroSuppressionHLT
+
+from RecoLocalTracker.SiStripClusterizer.DefaultClusterizer_cff import *
+siStripClustersHLT = cms.EDProducer("SiStripClusterizer",
+                                    Clusterizer = DefaultClusterizer,
+                                    DigiProducersList = cms.VInputTag(
+                                        ('siStripDigisHLT','ZeroSuppressed'),
```

Comment:
I think that you should keep defining them as `cms.InputTag` because you are not cloning them from some already defined parameter... But honestly I don't know if having defined the whole vector as `cms.VInputTag`. If you already checked that it works as such, it is also fine with me.